tasks/update-granules-cmr-metadata-file-links/package.json

```diff
@@ -40,16 +40,16 @@
   "author": "Cumulus Authors",
   "license": "Apache-2.0",
   "dependencies": {
-    "@cumulus/aws-client": "18.5.3",
-    "@cumulus/cmrjs": "18.5.3",
-    "@cumulus/common": "18.5.3",
+    "@cumulus/aws-client": "18.5.4",
+    "@cumulus/cmrjs": "18.5.4",
+    "@cumulus/common": "18.5.4",
     "@cumulus/cumulus-message-adapter-js": "2.2.0",
-    "@cumulus/distribution-utils": "18.5.3",
-    "@cumulus/logger": "18.5.3",
+    "@cumulus/distribution-utils": "18.5.4",
+    "@cumulus/logger": "18.5.4",
     "lodash": "^4.17.15"
   },
   "devDependencies": {
     "@cumulus/aws-client": "18.5.3",
```

Comment:
This one didn't update for some reason?

Reply:
not sure why this is missed and the `Verify Lerna` command didn't catch this either.

Reply:
There is a duplicate entry "@cumulus/aws-client" in dependencies. We should remove this line.